### PR TITLE
PP9-13048 - Content Picker. No message for the user that he has permission limits for sharing content

### PR DESCRIPTION
**Pull request by tc4rdpp**
- API updated
 - success color (green) added to the theme file
 - access original toggle added
 - Access Original Content Rights validation
 - Shared Content Dialog component code cleanup
 - Notification component reworked
 - Dialog Base component refactored
 - Notification service refactored

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/dialog-base/dialog-base.component.ts [hunk 1]
```diff
@@ -44,4 +38,8 @@ export class DialogBaseComponent extends BaseComponent implements OnInit, OnDest
       this.notification = notification;
     });
   }
+
+  ngOnDestroy() {
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/dialog-base/dialog-base.component.ts:42]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/services/notification/notification.service.ts [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
 
-  constructor() {
-    // NOTIFICATION BEHAVIOR SUBJECT INITIALIZATION
-    this.notificationStore = {
-      message: '',
-      displayTime: 0,
-      status: false,
-      type: 'success',
-    };
+  notification = new BehaviorSubject<Notification>(this.notificationStore);
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/services/notification/notification.service.ts:17]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.ts [hunk 3]
```diff
@@ -179,33 +161,30 @@ export class ShareContentDialogComponent extends DialogBaseComponent implements
...
-      setTimeout(() => {
-        // SET NOTIFICATION PROPERTIES
-        this.notificationMessage = `#${response.referenceId} ${this.translatePipe.transform(
-          'ShareContentDialog.SuccessNotification'
-        )}`;
-        this.notificationType = 'success';
-        this.notificationStatus = true;
-        this.notificationDisplayTime = 10000;
-      }, 200);
+      const success = {
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.ts:164]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<div class="dialog-container" #contentContainer>
+<div *ngIf="sharedContentForm$ | async as sharedContentForm" class="dialog-container" #contentContainer>
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.html:1]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.scss [hunk 5]
```diff
@@ -287,3 +287,13 @@
   max-height: 80vh;
   overflow-y: auto;
 }
+
+/* Works on Firefox */
```
[the remark this answers is not in the document]
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/share-content-dialog/share-content-dialog.component.scss]: Done

**Review by rjs-picturepark — APPROVED** [review]

**Pull request merged**